src/plugins/discover/public/application/embeddable/saved_search_embeddable.tsx

```diff
@@ -168,6 +168,14 @@ export class SavedSearchEmbeddable
     this.searchProps!.isLoading = true;
 
     this.updateOutput({ loading: true, error: undefined });
+    const executionContext = {
```

Comment:
we don't get this context for saved searches in `Disover` app

Reply:
it's possible, but needs refactoring, could do this in a follow up PR

Reply:
it would be awesome 🔥